packages/ui/react-ui/src/Button.tsx

```diff
@@ -16,6 +16,7 @@ export const Button: FC<ButtonProps> = (props) => {
         <button
             className={`wallet-adapter-button ${props.className || ''}`}
             disabled={props.disabled}
+            style={...props.style}
```

Comment:
Hmm, is this right? Wouldn't it be either `style={{...props.style}}` or just `style={props.style}`?

Reply:
Definitely. The correct is style={{...props.style}}. I saw that after submitting the pull request, but didn't know if I should send another commit right next.

What is the best practice in that case? Should I have sent another commit fixing my error or should I have waited you review the change like I did?

Reply:
Always better to fix it before review if you find it first!

Reply:
I'm not sure why we can't just pass `style={props.style}` though. No reason to spread (copy) the object.

Reply:
Sure, let's use `style={props.style}` then.